src/utils/experimentUpdatesHandler.js

```diff
@@ -63,16 +65,43 @@ const onQCUpdate = (update, dispatch, experimentId) => {
 };
 
 const onGEM2SUpdate = (update, dispatch, experimentId) => {
-  const processingConfig = update?.item?.processingConfig;
+  const { response: { error, errorCode, userMessage } } = update;
+
+  if (error) {
+    console.log(errorCode, userMessage);
```

Comment:
if you want to print the error for debugging, better use `console.error`

Reply:
Oh right, changed